persistence/build.gradle

```diff
@@ -45,6 +45,10 @@ android {
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
+
+    lintOptions {
+        abortOnError false
+    }
```

Comment:
Without this, we would need to fix ~ 30 lint errors here:

```
ret += deleteDataForMeasurement(database, selectedMeasurementsCursor
                                        .getLong(selectedMeasurementsCursor.getColumnIndex(BaseColumns._ID)));
```

=> getColumnIndex can be -1 when the column does not exist, but `cursor.getLong()` only accepts an index int > 0

I'm not sure if we want to do much "cleanup" work in the SDK 6 fork, so I added the "ignore lint" option instead.